wallet-android/src/main/java/com/iohk/jormungandrwallet/Settings.kt

```diff
@@ -0,0 +1,21 @@
+package com.iohk.jormungandrwallet
+
+class Settings {
+    private external fun delete(settings: Long)
+    companion object {
+        init {
+            System.loadLibrary("wallet_jni")
+        }
+    }
+
+    private var settings_prt : Long = 0
+
+    constructor(ptr : Long) {
+        this.settings_prt = ptr
+    }
```

Comment:
not keen on exposing this constructor publicly... this can go wrong and we could end up with issues.

Reply:
Definetly. Good catch.

Reply:
Uhm, I am not sure how to make this actually. Since if making it private it cannot be used from `Wallet`